src/components/MainPanel/ExplorePanel.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import GifListView from "./GifListView";
+import SearchBar from './SearchBar';
+import ErrorBoundary from "../ErrorBoundary";
+import { bindActionCreators } from "redux";
+import actions from "../../actions";
+import { connect } from "react-redux";
+
+const { getTrendingGifs } = actions.creators;
+
+class ExplorePanel extends Component {
+
+  componentDidMount() {
+    this.props.getTrendingGifs();
+  }
+
+  render() {
+    return (
+      <div>
+        <SearchBar />
+        <ErrorBoundary message="Unable to load.">
```

Comment:
The message attribute value `Unable to load` may have not the most accurate content as this component will show on the tree despite having an error thrown or not.